workflow_calcium_imaging/reference.py

```diff
@@ -0,0 +1,22 @@
+import datajoint as dj
+
+from workflow_calcium_imaging import db_prefix
+
+schema = dj.schema(db_prefix + "reference")
+
+
+@schema
+class Equipment(dj.Lookup):
+    definition = """
+    scanner: varchar(32)
+    """
+    contents = zip(["ScannerA"])
+
+
+@schema
```

Comment:
Should we pull from the new element-animal surgery schema instead? [BrainRegion table](https://github.com/datajoint/element-animal/blob/main/element_animal/surgery.py#L77-L90)